.gitpod.yml

```diff
@@ -0,0 +1,7 @@
+image:
```

Comment:
Thinking about this further, let's split this into a separate PRs, please. We generally do atomic PRs with a particular or at least related changes - these are unrelated and should  probably be considered independent from another.

Reply:
fixed by @gnunicorn in https://github.com/TeamFranka/affinity/commit/c6abefd546d703dd1bdc0af9e78839776e9170d3